.github/workflows/ci.yml

```diff
@@ -66,7 +66,6 @@ jobs:
             ${{ matrix.os }}-${{ matrix.rust }}-${{ env.cache-name }}-
             ${{ matrix.os }}-${{ matrix.rust }}-
             ${{ matrix.os }}-
-
```

Comment:
Why are some newlines removed and some kept ?

Reply:
That's a good question. I'll reduce the noise of the commit.

Reply:
Unless you favor one way over the other and want me to unify.